src/raftstore/store/worker/pd.rs

```diff
@@ -696,10 +694,9 @@ impl<T: PdClient> Runnable<Task> for Runner<T> {
                     peer_stat.last_read_bytes = peer_stat.read_bytes;
                     peer_stat.last_read_keys = peer_stat.read_keys;
                     peer_stat.last_report_ts = time_now_sec();
-                    last_report_ts = cmp::max(
-                        last_report_ts,
-                        peer_stat.last_report_ts - self.region_heartbeat_interval,
-                    );
+                    if last_report_ts == 0 {
+                        last_report_ts = self.start_ts;
```

Comment:
The first time of heartbeat the `last_report_ts` will be set to `self.start_ts` and will keep this value always, I think it is not correct.

Reply:
The local variable 'last_report_ts' is set to the 'peer_stat.last_report_ts' at first, while the 'peer_stat.last_report_ts' will be updated as the current timestamp in every heartbeat, therefore make the local variable 'last_report_ts' changed.